test/CMakeLists.txt

```
@@ -775,6 +774,16 @@ add_custom_test(test_conv_igemm_mlir_xdlops_fwd SKIP_UNLESS_ALL HALF_ENABLED INT
     COMMAND ${IMPLICITGEMM_MLIR_ENV_F_XDLOPS} $<TARGET_FILE:test_conv2d> ${IMPLICITGEMM_MLIR_ARGS_F} --input 256 256  56 56 --weights 256  64   1 1 --pads_strides_dilations 0 0 1 1 1 1 --group-count 4
 )
 
+if(MIOPEN_USE_COMPOSABLEKERNEL)
+add_custom_test(test_conv_hip_igemm_xdlops SKIP_UNLESS_ALL HIP_ENABLED HALF_DISABLED FLOAT_DISABLED INT8_ENABLED GFX900_DISABLED GFX906_DISABLED 
```

```suggestion
add_custom_test(test_conv_hip_igemm_xdlops SKIP_UNLESS_ALL OCL_DISABLED HALF_DISABLED FLOAT_DISABLED INT8_ENABLED GFX900_DISABLED GFX906_DISABLED GFX90A_DISABLED
```
HIP and OCL are enabled by default -- no need to enable HIP but we must disable OCL. GFX90A is enabled by default and must be disabled.

---

Resolved!